indico/htdocs/js/indico/modules/sessions.js

```diff
@@ -0,0 +1,149 @@
+(function(global) {
+    'use strict';
+
+    function handleRowSelection() {
```

Comment:
Check on the commits are going to be merged from my **devel** branch. I just added `enableIfChecked` that handles this almost transparently.